docs/user-guide/deploying-your-docs.md

```diff
@@ -30,27 +30,29 @@ That's it! Behind the scenes, MkDocs will build your docs and use the
 Use `mkdocs gh-deploy --help` to get a full list of options available for the
 `gh-deploy` command.
 
-Be aware that you will not be able to review the built site before it is pushed
-to GitHub. Therefore, you may want to verify any changes you make to the docs
-beforehand by using the `build` or `serve` commands and reviewing the built
-files locally.
+Please note that you will not be able to review the built site before it is
+deployed to GitHub. Instead, you may want to verify any changes you made by
+using the `build` or `serve` commands and reviewing the built files locally
+before running `gh-deploy`.
 
 WARNING:
-You should never edit files in your pages repository by hand if you're using
-the `gh-deploy` command because you will lose your work the next time you
-run the command.
+The site files in the `gh-pages` branch are **automatically generated** and
+updated when using the `gh-deploy` script.  
+As such, you should never make any changes to the site files of your GitHub
+Pages site by hand. Doing so will cause you to lose these changes the next time
+the `gh-deploy` script or command is called.
 
 WARNING:
 If there are untracked files or uncommitted work in the local repository where
-`mkdocs gh-deploy` is run, these will be included in the pages that are deployed.
+`mkdocs gh-deploy` is run, these will be included in the deployed Pages site.
 
-### Organization and User Pages
+### User and Organization Pages
```

Comment:
If this is not strictly necessary, I think it's preferable not to change the heading, as it changes the anchor and could therefore soft-break links (soft because still linking to the right page, but to the top of the page and not the heading anymore).

Reply:
Good point! I'll get you a fix for this.